src/pages/Donations/Table.tsx

```diff
@@ -100,35 +100,39 @@ export default function Table({
               <Link
                 to={`${
                   appRoutes[
-                    row.chainId === chainIds.juno ? "profile" : "marketplace"
+                    row.viaId === chainIds.juno ? "profile" : "marketplace"
                   ]
                 }/${row.id}`}
```

Comment:
This now results in corrupt URLs:
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/2aac11cc-23fc-40e0-a01b-ad696455d83c)

Should this be `row.recipientId`?

Reply:
fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2853/commits/826ff9942672caa2d88a5bdea265b3f81ca7e964